sdk/python/tensorflow_metadata/proto/v0/path_pb2.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: tensorflow_metadata/proto/v0/path.proto
-"""Generated protocol buffer code."""
```

Comment:
This is weird. Was this manually edited?

Reply:
Don't think I edited anything manually for a pb2 file
I can start from a fresh master and see if that happens again after compiling protos ...